share_extension/android/extension_post/extension_post.js

```diff
@@ -638,7 +638,6 @@ const getStyleSheet = makeStyleSheetFromTheme((theme) => {
             width: 38,
         },
         video: {
-            backgroundColor: theme.centerChannelBg,
```

Comment:
I think this can be safely removed since we have parent element that sets `backgroundColor`, like the `fileContainer` and `container`.